compiler/tests-spec/testData/diagnostics/linked/expressions/constant-literals/character-literals/p-4/neg/1.1.fir.kt

```diff
@@ -5,13 +5,13 @@
 // TESTCASE NUMBER: 1
 
 fun case1() {
-    //less then four hex digits
+    //less than four hex digits
     val c0 = <!ILLEGAL_CONST_EXPRESSION!>'\u'<!>
     val c1 = <!ILLEGAL_CONST_EXPRESSION!>'\uf'<!>
     val c2 = <!ILLEGAL_CONST_EXPRESSION!>'\u1f'<!>
     val c3 = <!ILLEGAL_CONST_EXPRESSION!>'\u1wf'<!>
 
-    //more then four hex digits
+    //more than four hex digits
```

Comment:
If you're fixing this here in `1.1.fir.kt`, please also do the same change for `1.1.kt`, otherwise the test will fail :) (`x.kt` and `x.fir.kt` should have the same sources modulo diagnostics)

Reply:
I'm afraid I don't understand. The comment typo 'less then' -> 'less than' only occurred in `.../character-literals/p-4/neg/1.1.fir.kt`. The spelling in the associated `1.1.kt` was correct. I'm probably missing something (new to Kotlin, and this codebase!)

Reply:
No, here's `1.1.kt` in your branch, it says "more then": https://github.com/JetBrains/kotlin/blob/f280c0cf8216f659cedbfa373d06e982e5521d6a/compiler/tests-spec/testData/diagnostics/linked/expressions/constant-literals/character-literals/p-4/neg/1.1.kt#L26
while in `1.1.fir.kt` (this line) you've fixed it to "than".

I'm less concerned about the typos or your fixes, and more about the fact that sources of `1.1.kt` and `1.1.fir.kt` should be the same, or the test would fail :).

Reply:
> I'm less concerned about the typos or your fixes
I would be too :-)

Turns out I'd forgotten to commit that `1.1.kt` change. Done now.